en/peripherals/gripper.md

```diff
@@ -0,0 +1,107 @@
+# Gripper Peripherals
+
+Grippers are mechanical devices that can be integrated with an unmanned vehicle to hold (grip) and release payloads.
+
+PX4 allows grippers to triggered automatically in [Payload Delivery Missions](../flying/package_delivery_mission.md) or manually using a Joystick.
+
+![High-load gripper example](../../assets/hardware/grippers/highload_gripper_example.jpg)
+
+:::note
```

Comment:
FYI new note. I realised that if used as a generic actuator a gripper is not a gripper. So this links to the actuator docs - if someoen wants to do that they know enough now to find it. The rest of this doc is specific to your solution.

Reply:
Right, I could add a section on using Gripper as a 'generic actuator', as that is probably the most common use case!

Reply:
Added in https://github.com/PX4/PX4-user_guide/pull/2072/commits/8cb6995af87a95eeadbb32f22a5dad2be1c2b1a2

Reply:
A gripper is a gripper (as you have documented). A generic actuator that happens to be mapped to a gripper is not a gripper :-). It's enough to have this note. I have tidied it up.